open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataTopicConnector.java

```diff
@@ -420,8 +420,8 @@ int getNumberOfUnprocessedEvents() {
         boolean getRunningBrokers(Properties connectionProperties ) {
 
             boolean found = false;
-            try  (AdminClient adminClient = KafkaAdminClient.create(connectionProperties))
-            {
+            AdminClient adminClient = KafkaAdminClient.create(connectionProperties)
```

Comment:
If adminClient is null ie the create fails, we will end up exiting the retry loop.
However I can't see any compelling failures that might lead to this so probably ok.
Are you comfortable with KafkaAdminClient behaviour?

Reply:
Handled NPE.

Reply:
Thanks